docs/quickstart.md

```diff
@@ -33,14 +33,22 @@ section below:
 Follow steps 1 and 2 of the [Google Provider Setup](google_provider_setup.md)
 documentation.
 
-**⚡️ Note:** Use `http://sso-auth.localtest.me/oauth2/callback` as the
+**⚡️ Note:** Use `http://sso-auth.localtest.me/google/callback` as the
 **Authorized redirect URI** in step 2.
 
 At the end of step 2, you will have a client ID and client secret. Create a new
-file called `env` with those values, like so:
+file called `env` and input those values like so:
 
-    CLIENT_ID=<client id value>.apps.googleusercontent.com
-    CLIENT_SECRET=<client secret value>
+    PROVIDER_GOOGLE_CLIENT_ID=<client id value>.apps.googleusercontent.com
+    PROVIDER_GOOGLE_CLIENT_SECRET=<client secret value>
+
+As well as these, you also need to give a provider 'slug' for the configuration of the
+provider http routes.
+
+Add the below lines into the `env` file you just created:
+
+    PROVIDER_GOOGLE_TYPE=google
+    PROVIDER_GOOGLE_SLUG=google
```

Comment:
Maybe we should make the identifier this slug instead of passing it in. Too magical?

Reply:
I like this idea. I don't personally feel like it's needed prior to the release (just in case anyone feels differently), but think it would be good to add!